central/src/main/webapp/components/ModelView.jsx

```diff
@@ -250,6 +252,9 @@ export default function ModelView(props) {
 							: <DynForm data={noSynset}/>
 						}
 					</TabPanel>
+					<TabPanel value={index} index={6} className={classes.tabpanel}>
+                        <ModelDownloadButtons modelName={props.model.name}/>
```

Comment:
Maybe it would be better to add a Menu with actions which are shown all the time instead of adding a tab with download action.
If no more interactions for models are planned this is fine. but I am just wondering in case more actions to be included in future, if we want to have 1 tab per action. 
ffrom a usability perspective it is usally a better approach to show all possible actions on a central place. 

As I said just a thought in case that more interactions with the model are planned.

Reply:
I like this idea actually. I wanted to change what the UI looked like, but thought it would be messy to add a change like that to what should be a simple PR. Also, I didn't want to delay a PR while the actual functionality hasn't been checked.

Reply:
I also am not completely sure of the end goal for this project, so it could be very much the plan to add more actionable items and if that's the case I think that you surely had a better solution with the action button.

Reply:
speaking of future plans for this sub-project, I was also thinking if it might be better to do the download through the central server instead of just sending the link to the web-client.

At the moment every web-client/browser needs access through firewalls/proxies to download the model from the original URL as well as to central.
Instead of  that it would be possible that just the central-server is configured to access all necessary network-resources.
The client could just call an URL of the central-server and the server would stream the model-download to the client 
 I hope you know what I mean :-) ?

This would open the door to some future improvements like:
1) Authentication/Authorisation to download managed by central
2) central could be used as a Mirror or for Caching of remote models - similar to how Nexus can be configured when using maven. Nexus is used often this way for caching or to control which artifacts are used inside of a company.
3) uploading models with central without granting access for all web-clients to whatever storage central is using. so different storage strategies would be possible.
4)  single point to configure firewall access. Would also be nice for cloud based services using models.

just a few ideas from a DevOp perspective 

But as I said depends on future plans for this project and can easily be added later on.

Reply:
I am having trouble thinking of another action that would go in a separate button. Maybe we make a more generic "Get Model" tab. Then, it could include the download button and some other actions about getting the model like a sample criteria snippet.

@ebamberg I think what you are saying about proxying the download through the model server makes sense (at least as an option). We can probably put that in a later PR, though. Do you want to open an issue about it?

Reply:
@zachgk yes sure I can open a issue about that. That sounds sensible. Can I do that on the project tab?

for other actions: I can think of "deploy model" (see serving, deploy to 1 or more remote serving instances) , "upload model", "create docker" , "remove model",  "invalidate model" to flag a model as "don't use anymore for new  projects" or so. just a few ideas, maybe some more fancy actions like "convert model" to convert a model from keras to TFLite for example.

Reply:
Github requires write permissions to add it on the project tab. If you go ahead and create a normal issue, I will add it to the project tab for you (feel free to add other issues too).

Reply:
> Github requires write permissions to add it on the project tab. If you go ahead and create a normal issue, I will add it to the project tab for you (feel free to add other issues too).

already done.
I have created a issue for this: https://github.com/awslabs/djl/issues/735